### PR TITLE
Add recipe for libcgroup 3.0.

### PR DESCRIPTION
**Pull request by maleadt**
This provides useful cgroup management utilities that I may use in PkgEval (instead of requiring systemd for `systemd-run`).

### CODE REVIEW

File: L/libcgroup/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+autoreconf -i
+
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-pam
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
```
Comment (giordano) [inline L/libcgroup/build_tarballs.jl:30]: `experimental` is no-op nowadays :slightly_smiling_face: 
```suggestion
platforms = supported_platforms()
```

Comment (maleadt) [reply L/libcgroup/build_tarballs.jl:30]: TIL! Do we still need the Julia 1.6 compat then?

Comment (giordano) [reply L/libcgroup/build_tarballs.jl]: Julia compat still defaults to 1.0, although we can consider switching it to 1.0 at some point.

Comment (giordano) [reply L/libcgroup/build_tarballs.jl]: On the other hand being explicit makes it easier to catch things like #5842 :slightly_smiling_face:

**Pull request merged**